### PR TITLE
Fix bugs related to the switch to CV5

### PR DESCRIPTION
**Pull request by civerachb-cpr**
Fix a couple of bugs that somehow never got caught in the previous release, enable ENU mode for the CV5 IMU now that the ros_mscl driver supports it.

### CODE REVIEW

File: heron_base/launch/base.launch [hunk 1]
```diff
@@ -32,23 +32,25 @@
     <group unless="$(optenv HERON_IMU_UM6 0)">
       <!-- newer herons use the microstrain CM5-25 IMU -->
       <group if="$(eval arg('namespace') == '')">
-        <include filename="$(find ros_mscl)/launch/microstrain.launch">
+        <include file="$(find ros_mscl)/launch/microstrain.launch">
           <arg name="name"          value="cv5" />
```
Comment (tonybaltovski) [inline heron_base/launch/base.launch]: Could name just be changed to `imu` to allow for any microstrain to be used?

File: heron_base/launch/base.launch [hunk 2]
```diff
@@ -32,23 +32,25 @@
...
           <arg name="port"          value="$(optenv HERON_IMU_PORT /dev/microstrain)" />
           <arg name="imu_frame_id"  value="imu_link" />
-          <remap from="cv5/data"    to="imu/raw_raw" />
```
Comment (tonybaltovski) [inline heron_base/launch/base.launch:39]: I believe if you did "~data", this remap would work.

**Review by tonybaltovski — CHANGES_REQUESTED** [review]

File: heron_base/launch/base.launch (hunk 1 above)
Comment (civerachb-cpr) [reply heron_base/launch/base.launch]: For practical purposes the name is purely cosmetic, so yeah. Changing it is no problem.  For the heron I believe they always use the CV5 as it's the easiest to waterproof.

File: heron_base/launch/base.launch (hunk 2 above)
Comment (civerachb-cpr) [reply heron_base/launch/base.launch:39]: I tried `~data`, `~/imu/data`, `imu/data`, `cv5/imu/data` and `/cv5/imu/data`.  None of them remapped the topic correctly.  I'm under the impression that the remap tag only works within `node`s and not within `include`s.

**Review by tonybaltovski — APPROVED** [review]

**Pull request merged**